### PR TITLE
More tests

### PR DESCRIPTION
**Pull request by mau11**
added tests and added some styling back to racer game

### CODE REVIEW

File: src/racer/common.css [hunk 1]
```diff
@@ -8,8 +8,8 @@ body          { font-family: Arial, Helvetica, sans-serif; }
 #controls     { width: 28em; float: left; padding: 1em; font-size: 0.7em; }
 #controls th  { text-align: right; vertical-align: middle; }
 #instructions { clear: left; float: left; width: 17em; padding: 1em; border: 1px solid black; box-shadow: 0 0 5px black; }
-#racer        { position: relative; z-index: 0; width: 1024px; height: 600px; margin-left: 20em; border: none; }
-#canvas       { position: absolute; z-index: 0; width: 1024px; height: 600px; z-index: 0; background-color: #72D7EE; }
+#racer        { position: relative; z-index: 0; width: 640px; height: 480px; margin-left: 20em; border: none; top: 60px; bottom: 20px; box-shadow: 0 0 20px #0d47a1}
```
Comment (stefanruijsenaars) [inline src/racer/common.css]: Did you make these smaller on purpose? (1024px -> 640px)

File: clientTests/e2e/client.js [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+    .click('link text', 'Log Out')
+    .pause(1000);
+  });
+
+  after(function(client, done) {
+    client.end(function() {
+      done();
+    });
+  });
+});
```
Comment (stefanruijsenaars) [inline clientTests/e2e/client.js:73]: Great stuff!

File: src/racer/common.css (hunk 1 above)
Comment (mau11) [reply src/racer/common.css]: I did, I was looking at a previous version and was trying to revert back to those changes. If it should be the larger values, I will change them back

Comment (mau11) [reply src/racer/common.css]: updated! :)

**Review by stefanruijsenaars — APPROVED** [review]

**Pull request merged**